src/style.css

```diff
@@ -0,0 +1,36 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
```

Comment:
last child should't have a margin. Use `:not(:last-child)`

Reply:
still not fixed